gdcdictionary/schemas/follow_up.yaml

```diff
@@ -1729,12 +1900,270 @@ properties:
       - Hepatitis C Virus RNA
       - Unknown
       - Not Reported
+    enumDef:
+      Unknown:
+        $ref:
+          - "_terms.yaml#/unknown/common"
+      Not Reported:
+        $ref:
+          - "_terms.yaml#/not_reported/common"
 
   weight:
     $ref:
       - "_terms.yaml#/weight/common"
     type: number
 
+  body_surface_area:
+    $ref:
+      - "_terms.yaml#/body_surface_area/common"
+    type: number
+
+  recist_targeted_regions_number:
+    $ref:
+      - "_terms.yaml#/recist_targeted_regions_number/common"
+    type: integer
+
+  recist_targeted_regions_sum:
+    $ref:
+      - "_terms.yaml#/recist_targeted_regions_sum/common"
+    type: number
+
+  adverse_event_grade:
+    $ref:
+      - "_terms.yaml#/adverse_event_grade/common"
+    enum:
+      - Grade 1
+      - Grade 2
+      - Grade 3
+      - Grade 4
+      - Grade 5
+
+  cd4_count:
+    $ref:
+      - "_terms.yaml#/cd4_count/common"
+    type: number
+
+  imaging_type:
+    $ref:
+      - "_terms.yaml#/imaging_type/common"
+    enum:
+      - MRI
+      - CT Scan
+      - PET
+      - 99mTc Bone Scintigraphy
+
+  scan_tracer_used:
+    $ref:
+      - "_terms.yaml#/scan_tracer_used/common"
+    enum:
+      - Axumin
+      - Choline
+      - Acetate
+      - PSMA
+      - Sodium Fluoride
+
+  nadir_cd4_count:
+    $ref:
+      - "_terms.yaml#/nadir_cd4_count/common"
+    type: number
+
+  hiv_viral_load:
+    $ref:
+      - "_terms.yaml#/hiv_viral_load/common"
+    type: number
+
+  aids_risk_factors:
+    $ref:
+      - "_terms.yaml#/aids_risk_factors/common"
+    enum:
+      - Candidiasis
+      - Cytomegalovirus
+      - Coccidioidomycosis
+      - Cryptococcosis
+      - Encephalopathy
+      - Herpes Simplex Virus
+      - Histoplasmosis
+      - Isosporiasis
+      - Mycobacterium avium complex
```

Comment:
Should 'complex' be capitalized?  I understand that "avium" and "tuberculosis" are probably species names right?   @tlicht3

Reply:
@wwysoc2 yes we can update this.